.github/workflows/build.yml

```diff
@@ -22,9 +22,16 @@ jobs:
           python-version: "3.6"
       - name: Setup Android SDK
         uses: android-actions/setup-android@v2
-      - name: Install Android SDK 23 and 26
+      - name: Install Android SDK 23
         run: |
           echo 'y' | sdkmanager --install "platforms;android-23"
+      - name: Setup Cache
+        uses: actions/cache@v2
+        with:
+          key: ${{ runner.os }}-buck
```

Comment:
This doesn't look right. Key should be unique for each configuration, e.g. it should hash all files. I'd rather disable caching completely as we don't have large enough volume of builds.

Reply:
removed caching config